src/Components/APICalls.js

```diff
@@ -4,15 +4,24 @@ const fiveMinutesMs = 1000 * 60 * 5;
 
 const apiUrl = `https://api-jet-lfoguxrv7q-uw.a.run.app`;
 
-export async function APISearch(inputValue) {
-  let response = await fetch(`${apiUrl}/anime/search?query=${inputValue}`, {
-    mode: "cors",
-  });
-  await handleErrors(response);
-  let responseJson = await response.json();
-  let temp = [];
-  responseJson.items.map((item, index) => temp.push(item));
-  return temp;
+export function useAPISearch(inputValue) {
+  return useQuery(
+    ["search:" + inputValue],
+    async () => {
+      let response = await fetch(`${apiUrl}/anime/search?query=${inputValue}`, {
+        mode: "cors",
+      });
+      await handleErrors(response);
+      let responseJson = await response.json();
+      let temp = [];
+      responseJson.items.map((item, index) => temp.push(item));
+      // Clears option field if search term is "".
+      if (inputValue.length === 0) return [];
```

Comment:
Do we want to return nothing on the 0-prefix (empty) search?  The api will return a list of the most popular anime, might be nice to have it show up as suggestions before the user types anything.

Reply:
I thought that was confusing...but it looks like YT/LinkedIn/Discord all agree with you, so I've incorporated it back in.